source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -57,3 +57,51 @@ todosController.get('remaining');
 
 Note that `@each` only works one level deep. You cannot use nested forms like
 `todos.@each.owner.name` or `todos.@each.owner.@each.name`.
+
+Sometimes, you may only care if the items of an array have changed, rather than their specific properties.
+For example, items may have been added to or removed from the array. In this case, you can create a
+computed property that relies on the leaf node:
+
+
+```app/controllers/todos.js
+export default Ember.Controller.extend({
+  todos: [
+    Ember.Object.create({ isDone: true }),
+    Ember.Object.create({ isDone: false }),
+    Ember.Object.create({ isDone: true })
+  ],
+
+  selectedTodo: null,
+  indexOfSelectedTodo: Ember.computed('selectedTodo', 'todos.[]', function () {
+    return this.get('todos').indexOf(this.get('selectedTodo'));
+  })
+});
+```
+
+Note here that we use the special key `[]` instead of `@each`, which is used for observing 
```

Comment:
`[]` is used for observing a specific dependent key on each item? (aka, can you touch up the phrasing? ;P)